composer.json

```diff
@@ -14,16 +14,15 @@
         "drupal-composer/drupal-scaffold": "~2.2",
         "drupal/config_devel": "~1.2",
         "drupal/console": "~1.0",
-        "drupal/drupal-extension": "~4.0",
+        "drupal/drupal-extension": "~4.0.0@beta",
         "drush/drush": "~9.0@stable",
+        "openeuropa/behat-transformation-context" : "~0.1",
         "openeuropa/code-review": "~1.0@beta",
         "openeuropa/drupal-core-require-dev": "^8.6",
-        "openeuropa/task-runner": "~1.0-beta4",
+        "openeuropa/task-runner": "~1.0-beta5",
         "phpunit/phpunit": "~6.0"
     },
     "scripts": {
-        "post-create-project-cmd": "./build-module.sh",
-        "drupal-scaffold": "DrupalComposer\\DrupalScaffold\\Plugin::scaffold",
```

Comment:
Why are these removed. They are both in the module template: https://github.com/openeuropa/drupal-module-template/blob/master/composer.json

@ademarco @voidtek @dxvargas should these be here ?